--- wxsvg-1.1.8/src/Makefile.am.link	2012-05-29 09:19:51.000000000 +0000
+++ wxsvg-1.1.8/src/Makefile.am	2012-05-29 09:20:12.000000000 +0000
@@ -46,7 +46,7 @@
   CSSValue.cpp CSSStyleDeclaration.cpp\
   SVGCanvas.cpp SVGCanvasItem.cpp NodeList.cpp $(LIBAV_SRC)
 
-libwxsvg_la_LIBADD = svgxml/libwxsvgxml.la $(RENDER_LIB) $(TEXT_LIB)
+libwxsvg_la_LIBADD = svgxml/libwxsvgxml.la $(RENDER_LIB) $(TEXT_LIB) -lpangocairo-1.0
 libwxsvg_la_LDFLAGS = -no-undefined -version-info @LIBRARY_VERSION@
 
 INCLUDES = -I$(top_builddir)/include -I$(top_builddir)/include/wxSVG
